test/integration/auth-methods/ntlm.test.js

```diff
@@ -0,0 +1,173 @@
+var expect = require('expect.js'),
+    http = require('http'),
+    ntlmUtil = require('httpntlm').ntlm;
+
+describe('NTLM', function () {
+    var ntlmServer,
+        ntlmServerPort = 3456,
+        testrun,
+        runOptions = {
+            collection: {
+                item: {
+                    name: 'NTLM Sample Request',
+                    request: {
+                        url: 'http://localhost:' + ntlmServerPort,
```

Comment:
Where is this server being started?

Reply:
in the `before` block [line 79](https://github.com/postmanlabs/postman-runtime/pull/390/files#diff-941f706d7c82f0f8633eb82a743ba94bR79)
and stopped in `after` block